install.sh

```diff
@@ -199,6 +199,28 @@ ops_install_qemu() {
   fi
 }
 
+ops_install_haxm() {
+    if [ "$OS" != "darwin" ]; then
+        return
+    fi
+
+    # check to see if a version of the package was previously
+    # installed or if the haxm kext was otherwise loaded. 
+    existing_install=`pkgutil --pkgs | grep haxm`
+    loaded_module=`kextstat | grep intelhaxm`
+    if [ ! -z $existing_install ] || [ ! -z $loaded_module ]; then
+        return
+    fi
+
+    # get the binary package. No need to check the md5 since the we are going
+    # for the included .dmg which is signed.
+    curl -LJO https://github.com/intel/haxm/releases/download/v7.4.1/haxm-macosx_v7_4_1.zip
+    tar -xvf haxm-macosx_v7_4_1.zip -C /tmp
+    hdiutil attach /tmp/IntelHAXM_7.4.1.dmg
+    installer -package /Volumes/IntelHAXM_7.4.1/IntelHAXM_7.4.1.mpkg -target /
```

Comment:
This requires permissions. It will fail as is.

Reply:
this is the part where a prompt to the user to indicate that they can get hardware acceleration if they sudo install this and letting the user gracefully opt-out if they so desire